blocks/bricks/interfaces.py

```diff
@@ -141,31 +141,61 @@ class Initializable(RNGMixin, Brick):
 
     """
     has_biases = True
```

Comment:
Backward compatibility is going to kill us, if we go for it. We have stable versions for those who work with their old models. So I think we should care only about getting a new solution for initialization.